agent/semgrep_agent.py

```diff
@@ -81,6 +104,9 @@ def process(self, message: m.Message) -> None:
         """
         content = message.data.get("content")
         path = message.data.get("path")
+        memory_limit = (
+            self.args.get("memory_limit", DEFAULT_MEMORY_LIMIT) or DEFAULT_MEMORY_LIMIT
+        )
```

Comment:
just `self.args.get("memory_limit", DEFAULT_MEMORY_LIMIT)`

Reply:
This for the case where the arg is exist and its value is None.